README.md

```diff
@@ -86,6 +86,13 @@ sharing.
 
 ### Installation:
 
+To run in standalone mode (http://localhost:8080):
```

Comment:
It could be worth documenting the purpose of this image, ie not for production and also documenting that this image does not run in development mode.

Reply:
good point. updated readme.